### PR TITLE
Add notice about rmw_connextdds

### PR DESCRIPTION
**Pull request by asorbini**
Add a paragraph to the beginning of README mentioning the availability of `rmw_connextdds`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 # rmw_connext
 Implementation of the ROS Middleware (rmw) Interface using [RTI's Connext DDS](https://www.rti.com). 
 
+**DEPRECATION NOTICE:** [rmw_connextdds](https://github.com/ros2/rmw_connextdds)
```
Comment (asorbini) [inline README.md:4]: No sure if "Deprecation Notice" is the correct term. Please feel free to suggest better wording, thanks!

Comment (GerardoPardo) [reply README.md:4]: Migration Notice?

Comment (clalancette) [reply README.md:4]: Deprecation Notice is fine with me.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+is a new RMW implementation for RTI Connext DDS, which supersedes the one
+contained in this repository (`rmw_connext_cpp`). This new implementation was
+developed by RTI in collaboration with the ROS 2 community, and it resolves
+several performance issue that were present in the previous implementation.
```
Comment (clalancette) [inline README.md:8]:
```suggestion
several performance issues that are present in this implementation.
```

File: README.md [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
+`rmw_connextdds` is already available in Rolling, and it will be included in
+future ROS 2 releases starting with Galactic.
```
Comment (clalancette) [inline README.md:10]: Thinking towards the future, I'd just say:

```suggestion
`rmw_connextdds` is included in ROS 2 releases starting with Galactic.
This rmw implementation will be supported until the end-of-life of the ROS distributions it is available in (ROS 2 Dashing and Foxy).
```

**Review by clalancette — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (asorbini) [reply README.md:4]: I'll leave it as "Deprecation Notice"

**Comment by asorbini** [comment]
Suggested changes applied, ready to be reviewed again.

**Pull request merged**